modules/installation-configuration-parameters.adoc

```diff
@@ -328,8 +336,8 @@ control plane MachinePool.
 |Valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region], such as `us-east-1`.
 
 |`platform.aws.amiID`
```

Comment:
I noticed that `compute.platform.aws.amiID` , `controlPlane.platform.aws.amiID` and `platform.aws.amiID` are all under `Optional AWS parameters` table, but either `compute.platform.aws.amiID`, `controlPlane.platform.aws.amiID` or `platform.aws.amiID` is required for usgov region.